Sources/Core/CompilerInfo.swift

```diff
@@ -1,5 +1,5 @@
 /// Describes the compiler; used as source of truth for conditional compilation.
-public struct CompilerInfo {
+public struct CompilerInfo: Codable {
```

Comment:
Can we avoid abbreviations?
```suggestion
public struct CompilerConfiguration: Codable {
```

Reply:
fixed in all places.